deploy/firesim

```diff
@@ -161,13 +146,18 @@ def runworkload(runtime_conf: RuntimeConfig) -> None:
 
 @register_task
 def buildafi(build_config_file: BuildConfigFile) -> None:
+    # shouldn't ever be able to get here because main() implements logging and redirecting to builtbitstream
```

Comment:
The only reason why it is there is so that it get's registered as a legal taskname for argparse to accept.  Otherwise, argparse will error if you give it and that's the same as not having the deprecation.  I could add to the comment to explain that.